components/gallery/GalleryItemToolBar.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="toolbar">
+    <NeoTooltip :label="$t('reload')" position="top">
+      <a no-shadow @click="handleReloadClick">
+        <NeoIcon
+          :icon="isLoading ? 'spinner-third' : 'arrow-rotate-left'"
+          size="medium"
+          :label="$t('reload')"
+          :spin="isLoading" />
+      </a>
+    </NeoTooltip>
+    <NeoTooltip :label="$t('fullscreen')" position="top">
+      <a no-shadow @click="$emit('toggle')">
+        <NeoIcon
+          icon="arrow-up-right-and-arrow-down-left-from-center"
+          size="medium" />
+      </a>
+    </NeoTooltip>
+    <NeoTooltip :label="$t('newTab')" position="top">
+      <a no-shadow @click="handleNewTab">
+        <NeoIcon icon="arrow-up-right" size="medium" />
+      </a>
+    </NeoTooltip>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { NeoIcon, NeoTooltip } from '@kodadot1/brick'
+
+import { useGalleryItem } from './useGalleryItem'
+import { MediaType } from '@/components/rmrk/types'
+import { resolveMedia } from '@/utils/gallery/media'
+
+type ReloadElement =
+  | HTMLIFrameElement
+  | HTMLVideoElement
+  | HTMLImageElement
+  | null
+
+const isLoading = ref(false)
+defineEmits(['toggle'])
+
```

Comment:
pls re-order this, should be 

imports

any types

define emits
define props
composables 
refs
computed 
methods